src/components/CodeEditor/CodeEditor.js

```diff
@@ -27,8 +27,6 @@ class CodeEditor extends Component {
   }
 
   componentDidMount() {
-    // Initial render() will always be a no-op,
-    // Because the mountNode ref won't exist yet.
     this._render();
```

Comment:
Technically this would still not work on first render if parents weren't already in the documented. I can add a setTimeout but don't care enough about this (?)